-------------------------------------------------------------------------------
Test set: AppTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.296 s <<< FAILURE! - in AppTest
initializationError(AppTest)  Time elapsed: 0 s  <<< ERROR!
org.mockito.exceptions.base.MockitoException: 


No tests found in AppTest
Is the method annotated with @Test?
Is the method public?

Caused by: java.lang.reflect.InvocationTargetException
Caused by: org.junit.runners.model.InitializationError

